render/hyperscript.js

```diff
@@ -5,6 +5,10 @@ var Vnode = require("../render/vnode")
 var selectorParser = /(?:(^|#|\.)([^#\.\[\]]+))|(\[(.+?)(?:\s*=\s*("|'|)((?:\\["'\]]|.)*?)\5)?\])/g
 var selectorCache = {}
 function hyperscript(selector) {
+	if (!selector || typeof selector !== "string" && !selector.view) {
+		throw Error("The selector should either be a string or an object with a view.");
```

Comment:
Try `"The selector must be either a string or component."`. That way, the message won't need edited once class support lands, and it's clearer on what types the constructor expects.